File: src/main/java/no/nav/fo/veilarbregistrering/oppgave/adapter/OppgaveResponseDto.java
package no.nav.fo.veilarbregistrering.oppgave.adapter;

import no.nav.fo.veilarbregistrering.oppgave.OppgaveResponse;

class OppgaveResponseDto implements OppgaveResponse {

    private long id;
    private String tildeltEnhetsnr;

    public OppgaveResponseDto(long id, String tildeltEnhetsnr) {
        this.id = id;
        this.tildeltEnhetsnr = tildeltEnhetsnr;
    }

    public OppgaveResponseDto() {
    }

    public long getId() {
        return this.id;
    }

    public String getTildeltEnhetsnr() {
        return this.tildeltEnhetsnr;
    }

    public void setId(long id) {
        this.id = id;
    }

    public void setTildeltEnhetsnr(String tildeltEnhetsnr) {
        this.tildeltEnhetsnr = tildeltEnhetsnr;
    }

}

File: src/main/kotlin/no/nav/fo/veilarbregistrering/oppgave/adapter/OppgaveResponseDto.kt
package no.nav.fo.veilarbregistrering.oppgave.adapter

import no.nav.fo.veilarbregistrering.oppgave.OppgaveResponse

internal data class OppgaveResponseDto(
    override val id: Long = 0,
    override val tildeltEnhetsnr: String? = null
) : OppgaveResponse